README.md

```diff
@@ -63,7 +71,7 @@ var ask = require('./ask.js');
 
 var answer = 'not empty';
 
-while (answer !== '' && answer !== 'SeCrEt') {
+while (answer !== (1<10)) {
```

Comment:
This line doesn't do what you think it does. 
`1 < 10` evaluates to `true`, so what you're really writing is
`while (answer !== true)`
which I don't think was your intention.